roadmap/roadmap.md

```diff
@@ -0,0 +1,90 @@
+# The Astropy Roadmap
+
+- [Status Legend](#Status-Legend)
+- [Functionality](#Functionality)
+- [Hardware and Performance](#Hardware-and-Performance)
+- [Learn and User Support](#Learn-and-User-Support)
+- [Community Building and Sustainability](#Community-Building-and-Sustainability)
+- [Infrastructure, Documentation](#Infrastructure-Documentation)
+- [Government, Management, and Personnel](#government-management-and-personnel)
+
+
+## Status Legend
+
+### Green :green_circle:
+- Group or person leading effort is identified and sufficiently supported.
+- Effort is already underway and/or expected to be implemented in the “near term.”
+
+### Orange :large_orange_diamond:    
+- Item is already defined (e.g., issue, APE exists), but not sufficiently underway.
+- Needs more people and/or resources to make it happen. 
+  - *For example, funding applied for, but not awarded.*
+  
+### Red :red_square:  
+- Consensus of the community is that the item is a priority.
+- Item won’t happen without more people/resources.
+  - *For example, there’s no one available to write a funding proposal or lead recruitment efforts.*
+  
+## Roadmap 2021 Project Board
+
+Issues relevant to the Roadmap goals can be added and organized on the [Astropy Roadmap 2021 Project Board](https://github.com/orgs/astropy/projects/7). The goal of the project board is to give a current overview of the status of our progress and provide access to the details of the work associated with roadmap items. It will be updated continuously.
+
+## Functionality
+
+- :large_orange_diamond: Provide next-generation spectroscopic reduction, analysis, and visualization tools usable by individual researchers and larger surveys.
+
+- :large_orange_diamond: Improve support and validation for input/output of standard file formats used in Astronomy including FITS, ASDF, HDF5, and ECSV as well as compression handling and enhanced support for large tables.
+
+- :large_orange_diamond: Improve interoperability and compatibility, while reducing duplication, amongst Astropy-affiliated packages and other community standard software.
+
+## Hardware and Performance
+
+- :large_orange_diamond: Improve Astropy ecosystem stability in High-Performance Computing environments, for example, and expand options for compatibility with read-only filesystems.

```

Comment:
I'm not sure what the “for example” means here.

Reply:
I think it's a typo, leftover text from a previous draft. I've removed it.